src/inscriptions/createRepeatInscriptions.ts

```diff
@@ -0,0 +1,20 @@
+import { getProviderOrThrow } from 'src/provider';
+import { CreateInscriptionOptions, CreateRepeatInscriptionsOptions } from './types';
+import { Json, createUnsecuredToken } from 'jsontokens';
+import { validateInscriptionPayload } from './utils';
+
+export const createRepeatInscriptions = async (options: CreateRepeatInscriptionsOptions) => {
+  const { getProvider } = options;
+  const provider = await getProviderOrThrow(getProvider);
+
+  validateInscriptionPayload(options.payload);
+
+  try {
+    const request = createUnsecuredToken(options.payload as unknown as Json);
+    const response = await provider.createInscription(request);
```

Comment:
is this supposed to call createInscription?

Reply:
ok I fixed it. even if the route will be the same on the extension, it will be more flexible to call the correct provider method and let the extension decide